dataplane/sdx/sdx.go

```diff
@@ -215,6 +215,22 @@ func RepairSdx(c *cli.Context) {
 	log.Infof("[RepairSdx] SDX cluster repair is started for: %s", name)
 }
 
+func formulateRequest(hostGroupToRepair string, hostGroupsToRepair []string) *sdxModel.SdxRepairRequest {
+	if len(hostGroupToRepair) == 0 && len(hostGroupsToRepair[0]) == 0 {
+		utils.LogErrorMessageAndExit(fmt.Sprintf("Please only specify either %s or %s", fl.FlHostGroupOptional.Name, fl.FlHostGroupsOptional.Name))
```

Comment:
The message is not consistent with the condition. There's no check for both being not null.

Reply:
Right, thanks, fixed.